website/templates/profile/notifications.mako

```diff
@@ -44,13 +45,23 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications" class="osf-treebeard-minimal">
+                    <div id="grid"></div>
+                    <div class="help-block" style="padding-left: 15px">
+                            <p id="configureNotificationsMessage"></p>
+                    </div>
+                </form>
```

Comment:
Sorry, I meant that having to use a submit button would be harder than the notification. Ideally, things that can be changed but require submit should have cancel buttons and dirty flags to warn users that they will lose changes. I would implement that (if it's not implemented) and keep this live rather than forcing consistency on a bad user interface pattern, with the ultimate goal of replacing out all of the submit buttons with live updates where possible.